__tests__/app.test.js

```diff
@@ -283,10 +283,89 @@ describe("Comments - POST/PATCH/DELETE", () => {
         })
     })
 })
-
-/*
- errors
- - invalid id type: ie string
- - article id doesnt exist
- - article id exists but no comments
- */
\ No newline at end of file
+describe("Votes", () => {
+    describe("PATCH - articles/:article_id", () => {
+        test("PATCH: 200 - responds with updated article", () => {
+            const voteIncrease = { inc_vote: 17 }
+            return request(app).patch("/api/articles/4")
+            .send(voteIncrease)
+            .expect(200)
+            .then(({body}) => {
+                const article = body.article
+                const keys = ["article_id", "title", "topic", "author", "body", "created_at", "votes", "article_img_url"]
+                expect(Object.keys(article)).toEqual(keys)
```

Comment:
This test is probably not necessary as the other test test that the keys are there. Also having this will limit you if you added more keys